### DISCUSSION TITLE
質問ページのTwitterカード表示のサポート

### DISCUSSION THREAD

**Discussion by simon-ritchie** [increments/qiita-discussions#44] [要望・アイデア・不具合]
現在Twitterで質問ページ（例 : https://qiita.com/mashirisu2/questions/2a1c5d201446fbc4ebc3 ）のURLを貼った時に、現在はカード表示がされない？ようですが通常の記事ページのようにカード表示がされると嬉しいです！

**Comment by kskwtnk** [comment]
@simon-ritchie 

ご意見ありがとうございます！

質問ページに関しては、カード表示以外も「記事では使えるのに質問では使えない」機能が現状多く存在してしまっています。
徐々に追いついていかせたい予定なのですが……工数の問題もありすぐに対応するのは難しいかもしれません 🙇 

ただ、認識はしていて改修予定はあることをまずは回答させていただきます！

**Comment by simon-ritchie** [comment]
@xrxoxcxox さん

承知いたしました！急ぎで無いと困る機能でもありませんので、のんびりと楽しみにお待ちしております・・・！

**Comment by kaitokimuraofficial** [comment]
@simon-ritchie
大変お待たせしました :bow: 

質問ページのOGP画像を設定しました！
質問ページのURLをX(旧 Twitter) などのSNS等で共有すると、OGP画像が表示されます。

詳細については以下のリリースノートをご覧ください。
https://qiita.com/release-notes/2024-07-24

**Reply by simon-ritchie** [reply]
@kaitokimuraofficial 

おお…！
ご対応ありがとうございます！！🎉🙇‍♂️
